src/gei/Commands/WaitForMigrationCommand.cs

```diff
@@ -60,6 +60,7 @@ public async Task Invoke(string migrationId, string githubTargetPat = null, bool
                 switch (state)
                 {
                     case RepositoryMigrationStatus.Failed:
+                    case RepositoryMigrationStatus.FailedValidation:
```

Comment:
can we use IsFailed() here? And same comment about checking IsPending explicitly and failing on unexpected state values.